grpc-example/grpc-example-scala/kubernetes/hello-impl.yml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: "apps/v1"
+kind: Deployment
+metadata:
+  name: "hello-impl-v1-0-snapshot"
+  labels:
+    appName: helloworld.GreeterService
+    appNameVersion: "hello-impl-v1-0-snapshot"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      appNameVersion: "hello-impl-v1-0-snapshot"
+  template:
+    metadata:
+      labels:
+        appName: helloworld.GreeterService
+        appNameVersion: "hello-impl-v1-0-snapshot"
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: hello-impl
+          image: "hello-impl:1.0-SNAPSHOT"
+          imagePullPolicy: IfNotPresent
+          env:
+            # normally APPLICATION_SECRET would be set from the k8s secret store
+            - name: APPLICATION_SECRET
+              value: "asdf1234"
+            - name: JAVA_OPTS
+              value: "-Dconfig.resource=prod-application.conf"
+          ports:
+            - containerPort: 11000
+              name: http
```

Comment:
We'll [need a port named `https`](https://github.com/lagom/lagom-samples/blob/1.6.x/grpc-example/grpc-example-scala/hello-proxy-impl/src/main/scala/com/example/helloproxy/impl/HelloProxyLoader.scala#L42).

I don't think the service on the port needs to be encrypted, though. The only limitation is that the DNS-SRV lookup includes the `https` as the port name. Of course, content is served on the port names `https` must be support at least HTTP/2.